docs/index.rst

```diff
@@ -785,6 +785,14 @@ The SecretStr and SecretBytes will be formatted as either `'**********'` or `''`
 
 (This script is complete, it should run "as is")
 
+Strict Types
```

Comment:
can you move the "StrictBool" section into here.

Reply:
Done.